packages/contracts-bedrock/scripts/go-ffi/bin.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import "os"
+
+func main() {
+	switch os.Args[1] {
+	case "diff":
+		DiffTestUtils()
+	case "trie":
+		FuzzTrie()
+	}
```

Comment:
default case with a panic to prevent silent misuse?

Reply:
Done